mobile/App.tsx

```diff
@@ -32,6 +33,13 @@ export default function App() {
           <Stack.Group screenOptions={{ presentation: 'modal' }}>
             <Stack.Screen name="Register" component={RegisterModal} />
           </Stack.Group>
+          <Stack.Group>
+            <Stack.Screen
+              name="LotteryDetails"
+              component={LotteryDetails}
+              options={options}
+            />
+          </Stack.Group>
```

Comment:
`LotteryDetails` screen doesn't need to be created inside of another `Stack.Group` since we're not providing any specific (custom) options for that screen. It can share the group with `Home` and `AddLottery` screens.

Reply:
Thanks! made the change!